lldb/include/lldb/Core/ValueObject.h

```diff
@@ -958,7 +958,7 @@ class ValueObject {
                                           int32_t synthetic_index);
 
   /// Should only be called by ValueObject::GetNumChildren().
-  virtual size_t CalculateNumChildren(uint32_t max = UINT32_MAX) = 0;
+  virtual uint64_t CalculateNumChildren(uint64_t max = UINT32_MAX) = 0;
```

Comment:
As long as we are changing things, do we want to pass in a `std::optional<uint64_t> max`? If not, then switch this to `UINT64_MAX`.

Reply:
Oh that's separately a good change to make apart from the return value I mentioned below.